src/main/webapp/globalVariables.js

```diff
@@ -18,6 +21,10 @@ let currentPage = 1;
 let next = true;
 // no of records to be shown on each page 
 let recordsPerPage = 3;
+// its not necessary that the no of records in a file will be a
+//  multiple of records per page so no of records on last page
+//  may vary from 0 to recordsPerPage
+let noOfRecordsOnLastPage = recordsPerPage;
```

Comment:
no we can't use size of data because if file size is greater than window size the length of data become constant and we start adding additional info in a way that it overrides the previous info so we may end up showing outdated  info